core/src/main/java/org/apache/accumulo/core/clientImpl/ResourceGroupOperationsImpl.java

```diff
@@ -109,7 +109,7 @@ public Map<String,String> getProperties(ResourceGroupId group)
       TVersionedProperties rgProps = ThriftClientTypes.CLIENT.execute(context,
           client -> client.getVersionedResourceGroupProperties(TraceUtil.traceInfo(),
               context.rpcCreds(), group.canonical()),
-          ResourceGroupPredicate.exact(group));
+          ResourceGroupPredicate.ANY);
```

Comment:
I think I understand the basis of this change to be:

You can ask any service what the config is for a RG group, because any of them can query ZK, regardless of what RG they are in.

Is that correct?

Reply:
Correct. To get the configuration, the runtime configuration, you need to connect to a server running in the resource group.